fvtest/compilertriltest/LongAndAsRotateTest.cpp

```diff
@@ -0,0 +1,494 @@
+/*******************************************************************************
+ * Copyright (c) 2017, 2017 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at http://eclipse.org/legal/epl-2.0
+ * or the Apache License, Version 2.0 which accompanies this distribution
+ * and is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the
+ * Eclipse Public License, v. 2.0 are satisfied: GNU General Public License,
+ * version 2 with the GNU Classpath Exception [1] and GNU General Public
+ * License, version 2 with the OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ *******************************************************************************/
+
+#include "JitTest.hpp"
+#include "jitbuilder_compiler.hpp"
+#include <gtest/gtest.h>
+#include <Jit.hpp>
+
+// these two vectors hold the value loaded by the iload node.
+std::vector<int32_t> i2lInput = {0x0,0x1,0x0000FFFF,0x2,0x3E,(int32_t)0x80000000,(int32_t)0xFFFF0000,0x40000000,0x7C000000,(int32_t)0xFFFFFFFF,0x0003C000,0x40000001};
```

Comment:
How about I just use a c style array then to avoid all confusion?

```int32_t i2lInput [] = {0x0,0x1,0x0000FFFF,0x2,0x3E,(int32_t)0x80000000,
                         (int32_t)0xFFFF0000,0x40000000,0x7C000000,
                         (int32_t)0xFFFFFFFF,0x0003C000,0x40000001};```

Reply:
Yep, that might work.